pycode/test/test_epidata_geoModificationGermany.py

```diff
@@ -49,9 +49,10 @@ class Test_geoModificationGermany(fake_filesystem_unittest.TestCase):
         '01001: 01', '11000: 11', '11001: 11', '05362: 05', '03452: 03',
         '01054: 01', '16077: 16'}
     statetocounty_string = {
-        '1: [', '2: [', '3: [', '4: [', '5: [', '6: [', '7: [',
-        '8: [', '9: [', '10: [', '11: [', '12: [', '13: [',
-        '14: [', '15: [', '16: ['}
+        '1: [1001,', '2: [2000], 3: [', '4: [', '5: [', '6: [', '7: [',
+        '8: [', '9: [', '10: [', '11: [', '12: [',
+        '13: [13003, 13004, 13071, 13072, 13073, 13074, 13075, 13076], 14: [', 
+        '15: [', '16: ['}
```

Comment:
Can you explain this change?

Reply:
This string tests the stateID to countyID map. I added some counties to prevent the test from returning False if the map doesnt contain any county IDs